docs/contribute/dev-setup.md

```diff
@@ -0,0 +1,112 @@
+---
+title: 'Dev setup'
+---
+
+To fix a bug or add a feature in [Storybook's monorepo](https://github.com/storybookjs/storybook), the first step is to set up your development environment.
+
+## Initial setup
+
+The following command downloads and builds the most recent development version of Storybook on your local machine:
+
+```sh
+git clone https://github.com/storybookjs/storybook.git
+cd storybook
+yarn bootstrap --core
+```
+
+> **_Note:_** On Windows, you may need to run `yarn` before `yarn bootstrap`!
```

Comment:
Interesting, it should be the same across all operating systems 🤔

Reply:
I haven't verified this, just copied it from `CONTRIBUTING.md`

Reply:
@shilman - @merceyz is right. I PR'd this recently here: https://github.com/storybookjs/storybook/pull/14707 . Not sure where that code went.